directions/forms/forms380.py

```diff
@@ -585,9 +585,9 @@ def printForm():
     printForm()
 
 
-def form_05(c: Canvas, dir: Napravleniya):
+def form_05(c: Canvas, dir_obj):
```

Comment:
Тип можно и оставить)
```suggestion
def form_05(c: Canvas, dir_obj: Napravleniya):
```